src/components/list/list.scss

```diff
@@ -93,9 +97,9 @@ $list-border-radius: 0.375rem; // 6px
         limel-icon {
             background-color: var(
                 --icon-background-color,
-                var(--lime-light-grey)
+                rgb(var(--contrast-700))
```

Comment:
I assume this is what made these icons a little lighter when not specifying any colours…

![image](https://user-images.githubusercontent.com/154725/114528352-3aeba700-9c49-11eb-8266-56d1aef79b4f.png)

I think they are a tad too light, making them look like they are disabled… when compared to the actually disabled item, it's pretty clear they're not, but still… and it's obviously worse if there are no disabled items, which is probably most often the case.

Reply:
💯 agree

Reply:
last fixup looks like this
![image](https://user-images.githubusercontent.com/35954987/114563388-a8123300-9c6f-11eb-8527-3a629c05b575.png)